.github/workflows/build.yml

```diff
@@ -54,6 +54,12 @@ jobs:
           docker run --rm -idt -p 5000:5000 rigetti/qvm -S
           docker run --rm -idt -p 5555:5555 rigetti/quilc -R
 
+      - name: Fetch Jupyter cache
```

Comment:
Just checking my understanding: this is both fetching and setting the cache, right?

Reply:
Yes. My understanding is that it simply populate that path in the container with the content that's in cache.